.circleci/build.sh

```diff
@@ -128,7 +128,6 @@ elif [[ "$CIRCLE_JOB" == "PYTORCH" ]]; then
     source venv/bin/activate
     git clone https://github.com/pytorch/pytorch.git --recursive
     cd pytorch
-    git checkout 7f86fb8995d9990fe1eac5a18335523cb3da7f09
```

Comment:
I'm curious how did we get a build failure on a master if we have not updated pytorch sha before?

Reply:
Because the diff that caused this was a pytorch that changed tensor types in pytorch and in glow, there was a corresponding pytorch github pr but not a corresponding glow github pr so no way for failure to occur on master. I removed this sha here because basically the thought is that if glow is tracking pytorch master in fbcode it should be in open source also other wise this will happen. If anyone has alternative thoughts I'm open to it though.

Reply:
Got it. I think this sounds good, it does not seem that build time increased a lot and anyway we track pytorch master in fbcode very close as you said.